installing/installing_bare_metal_assisted/installing-bare-metal-assisted.adoc

```diff
@@ -0,0 +1,18 @@
+:_content-type: ASSEMBLY
+[id="installing-bare-metal-assisted"]
+= Installing a cluster on bare metal using the Assisted Installer
+include::_attributes/common-attributes.adoc[]
+:context: installing-bare-metal-assisted
+
+toc::[]
+
+You can install {product-title} on bare metal using the {ai-full} (AI). The {ai-full} provides you with more options for customization than an installer-provisioned installation, while still running pre-flight validations and using smart defaults.
```

Comment:
If an abbreviation is introduced for the Assisted Installer, I'd use the expanded name of it in the next sentence.